src/components/sidebar/SidebarContent.tsx

```diff
@@ -0,0 +1,106 @@
+import { Box, Flex, Text } from "@chakra-ui/react"
+import _ from "lodash"
+import { ComponentProps, FC } from "react"
+import { AiFillHome, AiFillBank } from "react-icons/ai"
+import {
+	MdDocumentScanner,
+	MdCategory,
+	MdLocationCity,
+	MdReviews,
+	MdLocalOffer,
+} from "react-icons/md"
+import { GoSignOut } from "react-icons/go"
+import { IoLanguage, IoSettingsSharp } from "react-icons/io5"
+import { RiSecurePaymentFill } from "react-icons/ri"
+import { ImUserTie } from "react-icons/im"
+import { IconType } from "react-icons/lib"
+import { NavLink } from "react-router-dom"
+import { AuthRole } from "../../utils/enums"
+import { NavItem } from "./NavItem"
+import { SelectedNavItem } from "./SelectedNavItem"
+
+type IProps = ComponentProps<typeof Box> & { role: AuthRole }
+
+export const SidebarContent: FC<IProps> = ({ role, ...rest }) => {
+	const navItems = getNavItems(role)
+
+	return (
+		<Box
+			as="nav"
+			pos="fixed"
+			top="0"
+			left="0"
+			zIndex="sticky"
+			h="full"
+			pb="10"
+			overflowX="hidden"
+			overflowY="auto"
+			bg="gray.800"
+			borderColor="blackAlpha.300"
+			borderRightWidth="1px"
+			w="60"
+			{...rest}
+		>
+			<Box px="4" py="5">
+				<Text fontSize="2xl" ml="2" color="white" fontWeight="semibold">
+					CHECK IF LEGAL
+				</Text>
+				<Text ml="2" color="whiteAlpha.700">
+					{_.capitalize(role)} Portal
```

Comment:
can we do it in this way rather than using loadash?
`role.toUpperCase()
`

Reply:
it will capitalise all the letters while we wanted only first letter as capital